com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -336,14 +316,15 @@ internal Vector3 Scale
         /// </summary>
         [Tooltip("Sets whether this transform should sync in local space or in world space")]
         public bool InLocalSpace = false;
-        private bool m_LastInterpolateLocal = false; // was the last frame local
 
         /// <summary>
         /// When enabled (default) interpolation is applied and when disabled no interpolation is applied
-        /// Note: can be changed during runtime.
         /// </summary>
+        /// <remarks>
+        /// This should only be changed by the authoritative side during runtime. Non-authoritative changes
```

Comment:
Not sure I understand this comment, interpolation should only be applied on the ghost side, the authority never interpolates.

Reply:
Woops!
That comment was supposed to be for InLocalSpace!  
DOH! >.<